flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/utils/ArrayTypeTestBase.scala

```diff
@@ -19,7 +19,9 @@ package org.apache.flink.table.planner.expressions.utils
 
 import org.apache.flink.api.common.typeinfo.{BasicArrayTypeInfo, PrimitiveArrayTypeInfo, Types}
 import org.apache.flink.api.java.typeutils.{ObjectArrayTypeInfo, RowTypeInfo}
+import org.apache.flink.table.api.DataTypes
 import org.apache.flink.table.planner.utils.DateTimeTestUtil._
+import org.apache.flink.table.types.AbstractDataType
```

Comment:
Are this imports needed?  (IntelliJ seems to think "no", but I know that Scala can be tricky with imports!)

Reply:
good catch